src/views/CardSceneView.js

```diff
@@ -19,38 +19,36 @@ type Props = {
 };
 
 export default class SceneView extends React.PureComponent<Props> {
-
   constructor(props: DFSCardNavigatorProps) {
     super(props);
     (this: any)._trackState = this._trackState.bind(this);
   }
 
   _trackState(route, data) {
     if (!route.trackingPageName) {
-      console.warn(`Attribute \`trackingPageName\` not defined in routes for ${route.routeName}`);
+      console.warn(
+        `Attribute \`trackingPageName\` not defined in routes for ${route.routeName}`
+      );
     } else {
       this.props.trackingActions.trackState(route.trackingPageName, data);
     }
   }
 
-
   render() {
     const { routeProps, component: Component, scene } = this.props;
     const isActiveRoute = scene.isActive && !this.props.hasModal;
     return (
       <Component
         {...routeProps}
         key={scene.key}
-
         // routeKey used for ScreenFocusAware
         routeKey={scene.route.key}
         handleNavigate={this.props.handleNavigate}
         trackPage={data => this._trackState(scene.route, data)}
-        handleBack={this.props.handleBack}
-        isLeftSplitPaneComponent={this.props.isLeftSplitPaneComponent}
+        handleBack={this.props.handleBackAction}
```

Comment:
@caseycookie14 I know I added this but looks like a mistake, per #1 can you revert this change?

Also ` isLeftSplitPaneComponent={this.props.isLeftSplitPaneComponent}` should remain.